internal/website/content/howto/secrets/crypt.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Encrypt & Decrypt Data"
+date: 2019-03-21T17:44:28-07:00
+draft: true
+weight: 2
+---
+
+Once you have [opened a secrets keeper][] for the secrets provider you want,
+you can encrypt and decrypt small messages using the keeper.
+
+[opened a secrets keeper]: {{< ref "./open-keeper.md" >}}
+
+## Encrypting data
+
+To encrypt data with a keeper, you call `Encrypt` with the byte slice you
+want to encrypt.
+
+```go
+plainText := []byte("Secrets secrets...")
+cipherText, err := keeper.Encrypt(ctx, plainText)
+if err != nil {
+    return err
+}
+```
+
+## Decrypting data
+
+To decrypt data with a keeper, you call `Decrypt` with the byte slice you
+want to decrypt. This should be data that you obtained from a previous call
+to `Encrypt` with a keeper that uses the same secret material (e.g. two AWS
+KMS keepers created with the same customer master key ID). The `Decrypt`
+method will return an error if the input data did not come from the same
+secret keeper (**TODO(light)**: verify with @clausti and @shantuo that this
+is correct).
+
+```go
+var cipherText []byte // obtained from elsewhere and random-looking
+plainText, err := keeper.Decrypt(ctx, cipherText)
+if err != nil {
+    return err
+}
+```
+
+## Large Messages
+
+The secrets keeper API is designed to work with small messages (i.e. <10 KiB
+in length.) Cloud key management services are high enough latency that using
```

Comment:
Sentence reads a bit strangely to me. Something like "Cloud key management services are high latency and using them ..." is simpler IMHO